skops/io/tests/test_external.py

```diff
@@ -427,3 +428,56 @@ def test_quantile_forest(self, quantile_forest, regr_data, trusted, tree_method)
         assert_method_outputs_equal(estimator, loaded, X)
 
         visualize(dumped, trusted=trusted)
+
+
+class TestSciKeras:
+    """Tests for SciKerasRegressor and SciKerasClassifier"""
+
+    @pytest.fixture(autouse=True)
+    def capture_stdout(self):
+        # Mock print and rich.print so that running these tests with pytest -s
+        # does not spam stdout. Other, more common methods of suppressing
+        # printing to stdout don't seem to work, perhaps because of pytest.
+        with patch("builtins.print", Mock()), patch("rich.print", Mock()):
+            yield
+
+    @pytest.fixture(autouse=True)
+    def keras(self):
+        scikeras = pytest.importorskip("scikeras")
+        return scikeras
+
+    @pytest.fixture
+    def trusted(self):
+        return [
+            "scikeras.wrappers.KerasClassifier",
+            "keras.models.Sequential",
+            "keras.layers.core.Dense",
+            "keras.layers.core.Input",
+        ]
+
+    @pytest.fixture
+    def test_dumping_model(self, keras, trusted):
+        from scikeras.wrappers import KerasClassifier
+
+        # This simplifies the basic usage tutorial from https://adriangb.com/scikeras/stable/notebooks/Basic_Usage.html
+
+        def get_clf(meta):
+            n_features_in_ = meta["n_features_in_"]
+            model = keras.models.Sequential()
+            model.add(keras.layers.Input(shape=(n_features_in_,)))
+            model.add(keras.layers.Dense(1, activation="sigmoid"))
+            return model
+
+        clf = KerasClassifier(model=get_clf, loss="binary_crossentropy")
+
+        pipeline = Pipeline([("classifier", clf)])
+
+        dumps(clf, "keras-test.skops")
+        dumps(pipeline, "keras-test.skops")
+
+        X, y = make_classification(1000, 20, n_informative=10, random_state=0)
+        clf.fit(X, y)
+        dumped = dumps(clf, "keras-test.skops")
```

Comment:
```suggestion
        dumped = dumps(clf)
```

2nd argument to `dumps` is the compression level. Honestly, I'm surprised that this didn't raise an error.